Test variable PSSenderInfo exist

DomBros opened the pull request:
change 'Get-Variable -Name PSSenderInfo -ErrorAction SilentlyContinue' to 'Test-Path variable:PSSenderInfo' because when Set-StrictMode -version set to latest $error contains many errors which should be silently skkiped

DomBros committed:
Test variable PSSenderInfo exist

change 'Get-Variable -Name PSSenderInfo -ErrorAction SilentlyContinue' to 'Test-Path variable:PSSenderInfo' because when Set-StrictMode -version set to latest $error contains many errors which should be silently skkiped

mgr32 commented on PPoShTools/Public/Log/Write-Log.ps1:
```diff
@@ -131,7 +131,7 @@ Function Write-Log {
             $currentHostname = [system.environment]::MachineName
             $currentUsername = [System.Security.Principal.WindowsIdentity]::GetCurrent().Name
-            if (Get-Variable -Name PSSenderInfo -ErrorAction SilentlyContinue) {
+            if (Test-Path variable:PSSenderInfo) {
```
I would write Test-Path -Path variable:PSSenderInfo

mgr32 reviewed (changes requested):
Looks good, but please add '-Path'

DomBros committed:
added parameter Path

change 'Test-Path variable:PSSenderInfo' to 'Test-Path -Path Variable:PSSenderInfo'

DomBros commented:
added '-Path'
